app/controllers/resources_controller.rb

```diff
@@ -41,8 +41,9 @@ def register_params
     reg_params[:metadata_source] = col_catkey ? 'label' : 'symphony'
     reg_params[:other_id] = "symphony:#{col_catkey}" if col_catkey
     reg_params[:collection] = params[:structural][:isMemberOf]
-    # TODO: content_type (tag Process:Content Type:Book (ltr) ) can come from a lookup using params[:type]
-    # reg_param[:tags] = ['Process:Content Type:Book (ltr)']
+    # TODO: Once a release of cocina-models containing the `Cocina::Models::Vocab`
+    #       module has been cut (0.8.0), replace the URI string below
+    reg_params[:tag] = ['Process : Content Type : Book (ltr)'] if params[:type] == 'http://cocina.sul.stanford.edu/models/book.jsonld'
```

Comment:
Assuming I am on the right track, I could pull this into a new private method like so, if folks wish:

```ruby
private

def book_type?
  # TODO: Once a release of cocina-models containing the `Cocina::Models::Vocab`
  #       module has been cut (0.8.0), replace the URI string below
  params[:type] == 'http://cocina.sul.stanford.edu/models/book.jsonld'
end
```